workalendar/tests/test_usa.py

```diff
@@ -448,6 +448,13 @@ def test_state_year_2015(self):
         self.assertIn(date(2015, 3, 31), holidays)  # Cesar Chavez Day
         self.assertIn(date(2015, 11, 27), holidays)  # Thanksgiving Friday
 
+    def test_state_year_2019(self):
```

Comment:
I wonder why you have put this test in the California Test Case.
IMHO there should be a test in the "Core" USA test suite. You may leave this one below, but testing Thanksgiving day / Thanksgiving Friday here is a bit confusing.

Reply:
@brunobord I agree, I was also not very sure where to put Thanksgiving Friday tests. Since a few were places inside California, I modified that itself.
Yeah, go ahead with forking, anything which helps in fixing the code :)